api/src/diag/types.ts

```diff
@@ -92,11 +97,16 @@ export enum DiagLogLevel {
 
 /**
  * Defines options for ComponentLogger
+ *
+ * @since 1.0.0
  */
 export interface ComponentLoggerOptions {
   namespace: string;
 }
 
+/**
+ * @since 1.4.1
```

Comment:
Reviewer Note: this also showed up on a patch version. It was unintentionally left out of 1.4.0 - see https://github.com/open-telemetry/opentelemetry-js/releases/tag/api%2Fv1.4.1